test/functions/sungeun/procEqualsExternType.chpl

```diff
@@ -8,6 +8,8 @@ proc =(ref a: myExternRecord, b: int) {
 
 var ex0: myExternRecord;
 var ex1: myExternRecord;
+ex0;
+ex1; // avoiding copy-init
```

Comment:
Is this pattern due to a specific initialization-related feature being tested?
If not, please mention it in #14271 as "now users will need to do / be aware of ..."

Reply:
Yes, this test is checking `=` and so I was disabling the split initialization (because that would not run `=`).